action.yml

```
@@ -238,6 +238,10 @@ runs:
       if: ${{ inputs.phplinting == 'true' }}
       shell: bash
       run: |
+        if [[ -f vendor/bin/parallel-lint ]]; then
+          echo "Running parallel-lint"
+          vendor/bin/parallel-lint --exclude=vendor --exclude=.git .
```

```suggestion
          vendor/bin/parallel-lint --exclude vendor --exclude .git .
```
I tested this locally and it seems that you do not include the equals sign

---

Oh, well. That's silly of it lol.
Done.